### DISCUSSION TITLE
CVE-2022-40604

### DISCUSSION THREAD

**Discussion by lavsurgut** [apache/airflow#26859] [Q&A]
Hello guys, want to check with you if you know if this vulnerability is fixed and in which version. Is it only version 2.4?

**Comment by Taragolis** [comment]
> Hello guys, want to check with you if you know if this vulnerability is fixed and in which version

Usual if CVE fixed it also shown for which version it affected, you could check it within the public Vulnerability Database

- https://nvd.nist.gov/vuln/detail/CVE-2022-40604
- https://cve.report/CVE-2022-40604

> Is it only version 2.4?

Yes, any Airflow >= 2.4.0
